src/aof.c

```diff
@@ -1474,6 +1474,29 @@ struct client *createAOFClient(void) {
     return c;
 }
 
+int truncateAOF(char *filename, off_t valid_up_to) {
```

Comment:
```suggestion
int truncateAppendOnlyFile(char *filename, off_t valid_up_to) {
```
prefer to use AppendOnlyFile instead of AOF for function name